crypto/ffc/ffc_params_validate.c

```diff
@@ -13,6 +13,10 @@
  * It calls the same functions as the generation as the code is very similar.
  */
 
+#include <openssl/err.h>
+#include <openssl/bn.h>
+#include <openssl/dsaerr.h>
```

Comment:
These errors are a bit horrible to include here.. Should it just use ffc instead?

Reply:
There are no extra FFC error codes and I do not want to add them.